gnssanalysis/gn_io/trop.py

```diff
@@ -9,41 +9,70 @@
 from .. import gn_io as _gn_io
 
 
-def _read_tro_solution(path: str, recenter: bool = True) -> _pd.DataFrame:
+def _read_tro_solution(path: str, recenter: bool = True, trop_mode="Ginan") -> _pd.DataFrame:
+    """For backwards compatibility"""
```

Comment:
Docstring should be one-line sphinx (missing the description of input parameters, errors raise, what's returned, etc.: `:param` `:return`)
https://github.com/NilsJPWerner/autoDocstring/blob/master/docs/one-line-sphinx.md

Reply:
This function is up for deletion if no one uses it (as there is the `read_tro_solution` function below this